content/docs/use-cases/data-registries.md

```diff
@@ -41,56 +40,52 @@ _middleware_ between ML projects and cloud storage. Here are it's advantages:
 
 ## Building registries
 
-Data registries start like any other <abbr>DVC repository</abbr>, with
-`git init` and `dvc init`. A good way to organize their contents is by using
-different directories to group similar data, e.g. `images/`,
-`natural-language/`, etc. For example, our
-[dataset registry](https://github.com/iterative/dataset-registry) uses a
-directory for each part in our docs like `get-started/` and `use-cases/`.
-
 Adding datasets to a registry can be as simple as placing the data file or
-directory in question inside the <abbr>workspace</abbr>, and telling DVC to
-track it, with `dvc add`. For example:
+directory in question inside the <abbr>workspace</abbr>, and track it with
```

Comment:
I think the previous version was pretty good ... at least I don't see the current one superior? what is the reason changing this?

Reply:
I compressed 2 paragraphs and 2 code blocks into a single pair.